docs/api/covidcast-signals/quidel.md

```diff
@@ -68,60 +68,68 @@ $$
 p = \frac{100 x}{n}
 $$
 
-We estimate p across 3 temporal-spatial aggregation schemes:
+We estimate p across 6 temporal-spatial aggregation schemes:
+- daily, at the county level;
 - daily, at the MSA (metropolitan statistical area) level;
 - daily, at the HRR (hospital referral region) level;
-- daily, at the state level.
+- daily, at the state level;
+- daily, at the HHS level;
+- daily, at the US national level.
 
-**MSA and HRR levels**: In a given MSA or HRR, suppose $$N$$ COVID tests are taken
-in a certain time period, $$X$$ is the number of tests taken with positive
-results. 
+#### Standard Error
 
-For raw signals:
-- if $$N \geq 50$$, we simply use:
+We assume the estimates for each time point follow a binomial distribution. The
+estimated standard error then is:
 
 $$
-p = \frac{100 X}{N}
+\text{se} = 100 \sqrt{ \frac{\frac{p}{100}(1- \frac{p}{100})}{N} } 
 $$
 
-For smoothed signals, before taking the temporal pooling average,
-- if $$N \geq 50$$, we also use:
+#### Smoothing
+
+We add two kinds of smoothing to the smoothed signals:
+
+##### Temporal Smoothing
+Smoothed estimates are formed by pooling data over time. That is, daily, for
+each location, we first pool all data available in that location over the last 7
+days, and we then recompute everything described in the two subsections above. 
+
+The sample size provided for smoothed signals is different from that for raw signals. 
+The former one is re-definied to be the cumulative sample size in the rolling 7 day 
+window while the latter one is simply the sample size for that corresponding date. 
```

Comment:
Not needed -- "recompute everything described above" already includes sample size as defined here
```suggestion
```